### PR TITLE
feat: rename Blues Wireless (blues.io) to Blues (blues.com) and add CYGNET R4

### PR DESCRIPTION
**Pull request by rayozzie**

### CODE REVIEW

File: variants/STM32L4xx/L431C(B-C)(T-U)/variant_CYGNET_L4.cpp [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+  if (HAL_RCC_ClockConfig(&RCC_ClkInitStruct, FLASH_LATENCY_2) != HAL_OK) {
+    Error_Handler();
+  }
+  PeriphClkInit.PeriphClockSelection = RCC_PERIPHCLK_USB | RCC_PERIPHCLK_SDMMC1
+                                       | RCC_PERIPHCLK_ADC | RCC_PERIPHCLK_OSPI;
+  PeriphClkInit.AdcClockSelection = RCC_ADCCLKSOURCE_SYSCLK;
+  PeriphClkInit.OspiClockSelection = RCC_OSPICLKSOURCE_SYSCLK;
+  PeriphClkInit.UsbClockSelection = RCC_USBCLKSOURCE_MSI;
+  PeriphClkInit.Sdmmc1ClockSelection = RCC_SDMMC1CLKSOURCE_MSI;
```
Comment (fpistm) [inline variants/STM32L4xx/L431C(B-C)(T-U)/variant_CYGNET_L4.cpp:151]: No USB, no Ospi and no SDMMC

**Review by fpistm — CHANGES_REQUESTED** [review]
Hi @rayozzie 
Thanks for the PR.
I've made a branch on my fork with several updates:
https://github.com/stm32duino/Arduino_Core_STM32/compare/main...fpistm:Arduino_Core_STM32:cygnet

* split the new board support and the renaming.
* add generics L431Cx like this you have the linker script.
* fix clock config
* apply astyle and fix some formatting
* remove cygnet link as it does not exist at this time. It could be added later else CI is failed.

**Comment by rayozzie** [comment]
@fpistm thank you!  I had just started the porting work because our first hardware spin arrived this past week, and so my apologies for the rough edges.  We (myself and Zak) will exercise your branch ASAP and test the heck out of it.

Hope you're well; thank you again.

**Comment by fpistm** [comment]
> Hope you're well; thank you again.

Fine and welcome. Hope you are well too.
I've converted the issue as Draft, when ready let me know.

**Pull request closed without merging**